RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -325,11 +325,113 @@ void handlePreAdventure(location place)
 		}
 	}
 
+
```

Comment:
hey, you moved it to pre-adventure, love it.

Reply:
Yeah, It looks like it was only there for MP limiting casting.